packages/x-components/static-docs/build-search-ui/web-x-components-integration-guide.md

```diff
@@ -1,22 +1,23 @@
 ---
```

Comment:
We should change the name of the file to

`web-x-components-development-guide.md`

Reply:
Remember to change links pointing to it.

Reply:
Done!